static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx

```diff
@@ -12,13 +12,14 @@ describe('TeamIssuesBreakdown', () => {
       body: TestStubs.TeamIssuesBreakdown(),
     });
     const statuses = ['new', 'regressed', 'unignored'];
```

Comment:
These seem defined on the line above already

Reply:
Yeah.. I kinda gave up on this one bc typescript wants them as consts, but not as `readonly`. So I had to `as const` each element of the array

Reply:
Ahh, ok fair enough